### ISSUE TITLE
Upgrade

### ISSUE THREAD

**Issue by bingalls** [tinacms/tina-starter-grande#59]
## Description
Errors in JavaScript console

### Actual result
I finished updating my outstanding pull request from last month, which fixes all warnings.
This includes adding a 404 page.

### Remaining issues
* Broken external links & missing *.json storage files
* Did not update prettier, to minimize diffs
* Need to `eslint --init`
* Only updated tinacms lib to 0.26.0; More recent versions cause `UseCMS` errors
* Not yet updating to recent React 0.17

**Comment by DirtyF** [comment]
Hey @bingalls What's the issue here? How is it related to this starter?

**Comment by bingalls** [comment]
Yes, I am cleaning up all of the warnings, and upgrading.
Coming soon is a develop branch, for a future PR, which fixes eslint & prettier tests.
Note that the latest version of TinaCMS breaks this starter, but at least this PR gets it closer.

**Comment by DirtyF** [comment]
See https://tinacms.org/docs/releases/2020-08-03 for the upgrade instructions

**Comment by bingalls** [comment]
@DirtyF Thanks for the tip. I cannot seem to exploit it, however. While the docs mention v0.25, it no longer exists. v0.24 is as high as I can go, before putting in serious effort.
Note that `"tina": "0.26.0"` upgrade causes this error:
_Error: useCMS could not find an instance of CMS_
useCMS is part of a node module that is not mentioned in these upgrade instructions. As well, the listed instructions do not seem to be an impediment
I also pushed the develop branch in my repo, which fixes all prettier & eslint warnings.

**Comment by DirtyF** [comment]
@bingalls yeah sorry, I just realized this starter was developed and that Tina's API has changed a lot since, it's a much bigger task to upgrade it than fixing the eslint errors.

**Comment by bingalls** [comment]
After you merge my current Pull Request for the upgrades, I'll open another PR for my develop branch, with the eslint & prettier tests.
If there are docs on how I should apply these changes in Tina's API, I'd appreciate it.

**Comment by DirtyF** [comment]
@bingalls Scratch that this is the [gatsby plugin](https://github.com/tinacms/tinacms/tree/master/packages/gatsby-plugin-tinacms) that does embed the CMS, not the theme.

https://github.com/tinacms/tinacms/blob/master/packages/gatsby-plugin-tinacms/gatsby-ssr.tsx#L27-L32

**Comment by bingalls** [comment]
@DirtyF Please clarify this missing gatsby-ssr.tsx and the gatsby-plugin-tinacms package it belongs to.
While my version 0.24.0 package.json includes tinacms & gatsby-plugin-tinacms (both pinned together at that same v0.24), I have no Typescript files, and I cannot separate gatsby-plugin-tinacms from tinacms (is it a subrepo?). I also did not see your PR

**Comment by bingalls** [comment]
@DirtyF I signed the CLA, but Github does not seem to recognize this. PR is ready for review. Let me know, when you've added the Gatsby plugin PR
